### PR TITLE
Feature/issue 168/auto import to erlang

### LINKED ISSUE #168: Support auto-import BIF
The problem:

```erlang
f() ->
    abs(0).
```

```console
TODO:filename:-1: Unbound function: abs/1
```

### PR DESCRIPTION
**Pull request by yoshihiro503**
resolved #168

### CODE REVIEW

File: lib/bif.ml [hunk 1]
```diff
@@ -36,4 +36,6 @@ let type_sigs = [
      Type.(of_elem (TyFun ([Type.bool; Type.bool], Type.bool))));
     ({module_name="erlang"; function_name="not"; arity=1},
      Type.(of_elem (TyFun ([Type.bool], Type.bool))));
+    ({module_name="erlang"; function_name="abs"; arity=1},
```
Comment (amutake) [inline lib/bif.ml]: `abs` is not in PLT, right?

File: lib/context.ml [hunk 2]
```diff
@@ -20,11 +21,17 @@ let add key data t =
...
+    else
+      add (Key.MFA mfa) ty ctx
+  in
   Bif.type_sigs
-  |> List.fold_left ~f:(fun ctx (mfa, ty) ->
-                      add (Key.MFA mfa) ty ctx) ~init:ctx0
+  |> List.fold_left ~f:update ~init:ctx0
 
-let init () =
+let create ~import_modules =
```
Comment (amutake) [inline lib/context.ml]: 👍

File: lib/bif.ml (hunk 1 above)
Comment (yoshihiro503) [reply lib/bif.ml]: Right.
- https://github.com/dwango/fialyzer/issues/170
- https://github.com/erlang/otp/blob/OTP-21.1.4/lib/hipe/cerl/erl_bif_types.erl

Comment (amutake) [reply lib/bif.ml]: OK. Thank you for the links!

LGTM.

**Pull request merged**